package.json

```
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@embroider/addon-shim": "^0.50.2",
-    "ember-modifier": "^3.0.0"
+    "ember-modifier": "^3.2.7"
```

as ember-modifier@4.0.0 got published, let's include it in the range

```suggestion
    "ember-modifier": "^3.2.7 || ^4.0.0"
```

---

Done